cmd/root.go

```diff
@@ -0,0 +1,275 @@
+		)
+		platformPrompt.FilterPrompt = "   Platforms"
+		platformPrompt.FilterPlaceholder = "Find"
+		platformPrompt.PageSize = 5
+		choice, err := platformPrompt.RunPrompt()
+		if err != nil {
+			os.Exit(1)
+		}
+
+		platform := choice.String
+		fmt.Printf(" : Getting templates for %s...\n", platform)
+		templates, err := t.GetTemplates(platform)
+		if err != nil {
+			fmt.Println(" ! Failed to fetch templates :(")
+			os.Exit(1)
+		}
+
+		var templateChoices []string
+		for i := 0; i < len(templates); i++ {
+			templateChoices = append(templateChoices, templates[i].Name)
+		}
+
+		templatePrompt := selection.New(" > Choose a template:", selection.Choices(templateChoices))
+		templatePrompt.FilterPrompt = "   Templates for " + platform
+		templatePrompt.FilterPlaceholder = "Find"
+		templatePrompt.PageSize = 5
+		choice, err = templatePrompt.RunPrompt()
+		if err != nil {
+			os.Exit(1)
+		}
+
+		var selectedTemplate t.Template
+		for _, template := range templates {
+			if template.Name == choice.String {
+				selectedTemplate = template
+				break
+			}
+		}
+
+		fmt.Printf(
+			" : Downloading template assets from %s/%s...\n",
+			selectedTemplate.Owner, selectedTemplate.Repository,
+		)
+
+		tempDir, err := os.MkdirTemp("", "gmy-template-")
+		defer os.RemoveAll(tempDir)
+		if err != nil {
+			fmt.Println(" ! Failed to create temporary folder :(")
+			os.Exit(1)
+		}
+
+		downloadUrl, err := t.GetDownloadUrl(selectedTemplate)
+		if err != nil {
+			fmt.Println(" ! Failed to fetch template details :(")
+			os.Exit(1)
+		}
+
+		resp, err := grab.Get(tempDir, downloadUrl)
+		if err != nil {
+			fmt.Printf(" ! Failed to download assets from %s\n", downloadUrl)
+			os.Exit(1)
+		}
+		err = resp.Err()
+		if err != nil {
+			fmt.Printf(" ! Failed to download assets from %s\n", downloadUrl)
+			os.Exit(1)
+		}
+
+		fmt.Println(" + Assets downloaded")
+		fmt.Println(" : Setting up project folder...")
+
+		file, err := os.Open(resp.Filename)
+		if err != nil {
+			fmt.Println(" ! Failed to extract downloaded assets: ", err)
+			os.Exit(1)
+		}
+
+		err = files.Untar(projectName, file)
+		if err != nil {
+			fmt.Println(" ! Failed to extract downloaded assets: ", err)
+			os.Exit(1)
+		}
+
+		fmt.Printf(`
+ + Project folder set up successfully!
+   Additional configuration options for you:
+
+`)
+
+		gitPrompt := confirmation.New(" ? Initialize Git repository", confirmation.Yes)
+		initGit, _ := gitPrompt.RunPrompt()
+		if initGit {
+			err := utils.InitGit(projectName)
+			if err != nil {
+				fmt.Println(" - Failed to initialize Git repository.")
+			} else {
+				fmt.Println(" + Git repository initialized.")
+			}
+		}
+
+		if selectedTemplate.TSConfig_prompt {
+			tscPrompt := confirmation.New(" ? Would you like to add TypeScript configuration file", confirmation.No)
+			tsc, err := tscPrompt.RunPrompt()
+			if err != nil && tsc {
+				_, err := grab.Get(projectName, "https://raw.githubusercontent.com/grammyjs/cli/main/configs/tsconfig.json")
```

Comment:
`grammyjs/create-grammy`, not `grammyjs/cli`.

Since this cli only for scaffolding, thats why i choosed that name.